.github/workflows/sonarcloud.yml

```
@@ -33,19 +59,27 @@
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libpq-dev  # Required for PostgreSQL client
+          pip install psycopg2  # Install the PostgreSQL client in Python
+
+      # Install Python dependencies and run backend tests
+      - name: Install Python dependencies and run backend tests
+        run: |
+          cd backend
+          pip install -r requirements.txt
+          pip install pytest coverage
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)